.azure/gpu-tests-pytorch.yml

```diff
@@ -31,11 +32,11 @@ pr:
       - "src/pytorch_lightning/**"
       - "tests/tests_pytorch/**"
       - "setup.cfg"  # includes pytest config
+      - "setup.py"
       - "requirements/lite/**"
       - "src/lightning_lite/**"
-      - ".actions/**"
     exclude:
-      - "requirements/**/docs.txt"
```

Comment:
I suggest we keep the old pattern in case deeper directories are added. Otherwise this change needs to be reflected checkgroup.yml too

Reply:
ok, keeping one-level and if there is only one then set the name...